examples/convolution/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.13)
+project(convolution LANGUAGES CXX)
+
+find_package(Celerity 0.3.0 REQUIRED)
+
+add_executable(convolution convolution.cc)
+set_property(TARGET convolution PROPERTY CXX_STANDARD 17)
```

Comment:
General note: We should probably do this automatically in `add_celerity_to_target`. I've added it to *the list*.

Reply:
... but we need to make sure not to down-grade from C++20 if we do this.